### ISSUE TITLE
[QUESTION] [URGENT IF POSSIBLE] I have an app in production already

### ISSUE THREAD

**Issue by webpolis** [Digital-Sapphire/PyUpdater#220]
Hi

I was wondering how hashes are calculated, on which file? I see have a **versions.gz** having this:

```
        "win": {
          "file_hash": "fe4ee1ac72a7e9373e0118c1e60bf88d46f0ab3e91c457a64f03c7897a658453",
          "file_size": 24898006,
          "filename": "app-win-1.0.1.zip"
        }
```

Let's say I want that zip file to include an exe installer, so it's not going to be the _app.exe_ that runs the application. In that way, once it extracts and runs, it loads the installer again, and everything gets updated.

So, if I modify the zip that PyUpdater generates when I build the package using its CLI commands, I guess I also have to modify the hash. How would I handle this process?

Basically, I am modifying the packages (.tar.gz, zip, etc) for all platforms in a Makefile, and I want to update the hashes in the versions.gz file. I see you use the get_hash method to calculate a hash from a file. Is that from the tar.gz/zip version or the exe?

How would you recommend to proceed? I want the app to download an updated zip which includes the installer, not the app's binary.

Best regards,
Nicolas
